lib/media/track/localaudiotrack.js

```diff
@@ -80,6 +182,9 @@ class LocalAudioTrack extends LocalMediaAudioTrack {
    * @fires LocalAudioTrack#stopped
    */
   stop() {
+    // track was stopped, stop watching, and don't start watching again.
+    this._startWatching(null);
+    this._wasStopped = true;
```

Comment:
@makarandp0 Nitpick: Can we call this `_didCallStop`? We have another member named `didCallEnd` and it would be nice to have the naming consistent.